MantidQt/MantidWidgets/inc/MantidQtMantidWidgets/MuonFitPropertyBrowser.h

```diff
@@ -149,6 +190,32 @@ private slots:
   std::string m_simultaneousLabel;
   QtProperty *m_normalization;
   mutable QStringList m_normalizationValue;
```

Comment:
They're not changed inside `const` functions, the `mutable` keyword does not do anything here, it's only confusing